src/vs/workbench/browser/layout.ts

```diff
@@ -539,6 +539,48 @@ export class WorkbenchLayout implements IVerticalSashLayoutProvider, IHorizontal
 		return this.panelMaximized;
 	}
 
+	// Add reset for keyboard support
+	public resetSideBarWidth(): number {
```

Comment:
I originally thought of a third command to reset to defaults
I don't think that's necessary, need to remove